scripts/start.js

```diff
@@ -365,10 +376,29 @@ function getUserInfo () {
         return reject(err)
       }
       if (body['access_token']) {
-        resolve({
-          userName: username.slice(0, username.indexOf('@')),
-          ssoToken: body.access_token,
-          domain: domain
+        request(`${engineUrl}api/users`, { json: true, strictSSL: false, headers: { Authorization: `Bearer ${body['access_token']}` } }, (err2, response2, body2) => {
+          if (!err2 && body2.user && body2.user.length > 0 ) {
+            for (let i in body2.user) {
+              if (body2.user[i]['user_name'] === (username.slice(0, username.indexOf('@')) + '@' + domain)) {
+                resolve({
+                  userName: username.slice(0, username.indexOf('@')),
+                  ssoToken: body.access_token,
+                  domain: domain,
+                  userId: body2.user[i].id,
+                })
+                return
+              }
+            }
+          }
+          var userId = readlineSync.question(`oVirt user id (${DEFAULT_UUID}), can be found there ${engineUrl}api/users/ : `, {
```

Comment:
IIUIC, `(${DEFAULT_UUID})` always prints `(null)`, right?

If yes, what is the purpose of `DEFAULT_UUID` variable? Can't we just remove it, don't print 'null' for user and inline `null` at line 394?